paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -58,13 +58,27 @@ class CustomerSheet @Inject internal constructor(
         )
     }
 
+    fun configure(
+        configuration: Configuration,
+        intentConfiguration: IntentConfiguration,
```

Comment:
Is there a way to use this without intents? I thought CustomerSheet had a setup intent mode as well as another one.

Reply:
I should actually make `IntentConfiguration` default here. Most merchants probably aren't going to pass anything for `paymentMethodTypes`.